Reference serverless in patterns (#2000)
Qiskit/documentation: docs/guides/intro-to-patterns.mdx

## Guides demonstrating the Qiskit patterns workflow

### Qiskit Serverless

> [Qiskit Serverless](./serverless) allows users to deploy workloads to a cloud environments, allowing for local configuration and remote execution. Premium users have access to quantum-classical compute resources to build and deploy their own workloads remotely. Instances of Qiskit Serverless can also be deployed to other cloud environments. Check out the package's [documentation](https://qiskit.github.io/qiskit-serverless/index.html) for more information.

Reviewer: ```suggestion
With [Qiskit Serverless](./serverless), users can deploy workloads to cloud environments, configure them locally, and execute remotely. Premium users have access to quantum-classical compute resources to build and deploy their own workloads remotely. Instances of Qiskit Serverless can also be deployed to other cloud environments. Read the package's [documentation](https://qiskit.github.io/qiskit-serverless/index.html) for more information.
```
The IBM Style police tell us to avoid anthropomorphizing products (words like "let", "allow", "enable", etc.) so I tried a little rewrite.
